.ci.yaml

```
@@ -4441,3 +4441,33 @@ targets:
       tags: >
         ["devicelab", "hostonly"]
       task_name: windows_startup_test
+
+  - name: Windows flutter_tool_startup
```

```suggestion
  - name: Windows flutter_tool_startup__windows
```
I should catch this earlier: the devicelab target name follows `<platform> <task_name>`.

---

Done.